app/models/user.rb

```diff
@@ -42,35 +39,4 @@ class User < ActiveRecord::Base
             allow_nil: true
 
   has_secure_password
-
-  def remember
```

Comment:
**Code quality - Critical**

Slim the `UserModel` - moved the utility methods to service object.